fix: clarify test description for Use Rest Parameter with Function Parameter (#40725)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/use-the-rest-parameter-with-function-parameters.md

assert(sum() === 0);
```

> `sum` should be an arrow function and use the `...` rest parameter on the `args` parameter.

Reviewer: ```suggestion
`sum` should be an arrow function and use rest parameters syntax (`...`) on the `args` parameter.
```

> `sum` should be an arrow function and use rest parameters syntax (`...`) on the `args` parameter.

Reviewer 2: ```suggestion
`sum` should be an arrow function which uses rest parameter syntax (`...`) on the `args` parameter.
```

Total nitpick, but `rest parameter syntax` is not pluralised on MDN. Also, the wording feels a bit off, IMHO. How's this?

Author: It indeed sounds more natural. I thought about usage of plural form as well when @RandellDawson requested the previous change. The way I explained it to myself is that while it's put in a single place, it accepts multiple parameters and pluralization comes mainly from the latter fact. Although if consider only the former then singular form makes more sense. I don't know which approach is better. On the MDN page of `spread syntax` an opposite of it, is even referred to as just `rest syntax`.

To summarize I'll be happy with whichever is deemed here more appropriate. What do you think?

Reviewer 2: ![image](https://user-images.githubusercontent.com/63889819/105454519-e0890e80-5c36-11eb-8a2e-3488dc06be27.png)

I was basically following this grammatical approach. 🙂 

I'm not too set on one way or the other, though.

Reviewer 3: I agree the plural (_parameters_) sounds off. I would also argue the reason it should be singular is _it is **one**_ parameter. It just so happens to contain many parameters, but it itself is **a** parameter.

So, I would like the change suggested by Nicholas to be included.
